src/avfoundation.cs

```diff
@@ -2751,6 +2847,76 @@ interface AVAudioSession {
 		NSString SpatialPlaybackCapabilitiesChangedNotification { get; }
 	}
 
+	[NoMac]
+	[MacCatalyst (13, 1)]
+	enum AVAudioSessionCategory {
+		[Field ("AVAudioSessionCategoryAmbient")]
+		Ambient,
+
+		[Field ("AVAudioSessionCategorySoloAmbient")]
+		SoloAmbient,
+
+		[Field ("AVAudioSessionCategoryPlayback")]
+		Playback,
+
+		[Field ("AVAudioSessionCategoryRecord")]
+		Record,
+
+		[Field ("AVAudioSessionCategoryPlayAndRecord")]
+		PlayAndRecord,
+
+		[NoTV]
+		[NoWatch]
+		[Deprecated (PlatformName.iOS, 10, 0)]
+		[Deprecated (PlatformName.MacCatalyst, 13, 1)]
+		[Field ("AVAudioSessionCategoryAudioProcessing")]
+		AudioProcessing,
+
+		[Field ("AVAudioSessionCategoryMultiRoute")]
+		MultiRoute,
+	}
+
+	[NoMac] // Apple's documentation says the enum is available on macOS, but none of the individual values are, so just don't expose the enum on macOS.
+	enum AVAudioSessionMode {
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeDefault")]
+		Default,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeVoiceChat")]
+		VoiceChat,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeVideoRecording")]
+		VideoRecording,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeMeasurement")]
+		Measurement,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeGameChat")]
+		GameChat,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeMoviePlayback")]
+		MoviePlayback,
+
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeVideoChat")]
+		VideoChat,
+
+		[iOS (9, 0)]
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeSpokenAudio")]
+		SpokenAudio,
+
+		[Watch (5, 0), TV (12, 0), iOS (12, 0)]
+		[MacCatalyst (13, 1)]
+		[Field ("AVAudioSessionModeVoicePrompt")]
+		VoiceePrompt,
```

Comment:
```suggestion
		VoicePrompt,
```

Reply:
@haritha-mohan good catch, thanks! And fixed :)